docs/notebooks/pulse-efficient-transpilation.ipynb

```diff
@@ -594,16 +658,17 @@
      "output_type": "stream",
      "text": [
       "exact_value = -1\n",
-      "rzx_value = -0.6529376344086021+/-0.054665200205976434\n",
-      "ecr_value = -0.0647245283018868+/-0.048297550054371696\n"
+      "rzx_value = -0.4491175864977782+/-0.010477710362974534\n",
```

Comment:
My main concern here is that, it seems from the result of this run, the `rzx_value` is not too much closer to the `exact_value` compared to the `ecr_value`, which diminish the advantage of this method. Shall we check together if we have done the transpilation correctly and run it again?

Reply:
When I changed to optimization level = 1, I got this output: 

exact_value = -1
rzx_value = -0.3037473708400226 +/- 0.009251256033541494
ecr_value = -0.3011214445551086 +/- 0.010404926897466808